plugin/src/main/kotlin/tanvd/grazi/language/Lang.kt

```diff
@@ -20,8 +21,8 @@ enum class Lang(val displayName: String, val shortCode: String, private val clas
     RUSSIAN("Russian", "ru", "Russian", RemoteLangDescriptor.RUSSIAN, GraziBundle.langConfig("ru.rules.enabled")),
     PERSIAN("Persian", "fa", "Persian", RemoteLangDescriptor.PERSIAN),
     FRENCH("French", "fr", "French", RemoteLangDescriptor.FRENCH),
-    GERMANY_GERMAN("German (Germany)", "de", "GermanyGerman", RemoteLangDescriptor.GERMAN),
-    AUSTRIAN_GERMAN("German (Austria)", "de", "AustrianGerman", RemoteLangDescriptor.GERMAN),
+    GERMANY_GERMAN("German (Germany)", "de", "GermanyGerman", RemoteLangDescriptor.GERMAN, disabledCategories = GraziBundle.langConfig("de.categories.disabled")),
```

Comment:
Interesting, why do we need disabledCategories for DE now?

Reply:
Because we are use German in tests now, and typography grumbles at "" and others punctuation signs.